ods_ci/tasks/Resources/Provisioning/GPU/nfd_deploy.yaml

```diff
@@ -7,7 +7,8 @@ spec:
   instance: "" # instance is empty by default
   topologyupdater: false # False by default
   operand:
-    image: registry.redhat.io/openshift4/ose-node-feature-discovery:v4.11
+    # Image digest for registry.redhat.io/openshift4/ose-node-feature-discovery:v4.11
+    image: registry.redhat.io/openshift4/ose-node-feature-discovery@sha256:d6242132d2ddec00c46d22b63015a33af821eace0150ba47d185cd992fee317d
```

Comment:
so, for GPU operator we are going to install latest version available given that we are removing the `startingCSV` field, shouldn't we adopt the same approach with NFD and use some `latest` (if exists) or some other label?

Reply:
Hi @bdattoma , that's correct. I noticed that we update the channel but now I have noticed it is because of channel changes in `nfd` package among OCP versions :(

So, I will probably get back the `startingCSV` and update the disconnected automation.

Reply:
> we always install the latest version from the stable channel


yes. In my original comment I was referring to the NFD CustomResource, which here is fixed on a specific SHA. So I was wondering if we should use some "stable" or "latest" for the CR as well like we do for the operators

Reply:
>Currently, with this static approach we have to add the image egistry.redhat.io/openshift4/ose-node-feature-discovery@sha256:d6242132d2ddec00c46d22b63015a33af821eace0150ba47d185cd992fee317d explicitly into a disconnected env.


that raises the question: how much frequently would we update this image sha?